ui/desktop/app/routes/scopes/scope/projects.js

```diff
@@ -20,10 +20,9 @@ export default class ScopesScopeProjectsRoute extends Route {
    * @return {Promise
    */
   model() {
-    const currentScope = this.modelFor('scopes.scope');
-    return this.store.query('scope', {
-      scope_id: currentScope.id,
-    });
+    return this.store
```

Comment:
This could use an explanation because it's unclear why we would return _all_ projects when we're under scopes/scope route.  In desktop, we simply want to prime the store with all projects.  It may not even make sense to have this route any more now that we have recursive listings (this query can be made from scopes or scopes/scope).